samples/JwtBearerSample/Startup.cs

```diff
@@ -59,7 +57,7 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     OnAuthenticationFailed = c =>
                     {
-                        c.HandleResponse();
+                        c.Ignore();
```

Comment:
So we're back to the previous state where `HandleResponse()`/`SkipToNextMiddleware()` had multiple meanings? Oh please no, that was the whole point of my PR. You really hate me that much? :trollface: